be/src/vec/functions/function_datetime_floor_ceil.cpp

```diff
@@ -158,6 +158,9 @@ struct FloorCeilImpl {
     }
 };
 
+#define FLOOR 0
```

Comment:
It's better to use `constexpr` to avoid `define`

Reply:
good suggestion